tests/cppunit/config_test.cc

```diff
@@ -133,6 +133,33 @@ TEST(Config, GetAndSet) {
   }
 }
 
+TEST(Config, GetAndSetRenameCommand) {
+  const char *path = "test.conf";
+  unlink(path);
+
+  std::ostringstream string_stream;
+  string_stream << "rename-command KEYS KEYS_NEW"
+                << "\n";
+  string_stream << "rename-command GET GET_NEW"
+                << "\n";
+  string_stream << "rename-command SET SET_NEW"
+                << "\n";
+  std::ofstream output_file(path, std::ios::out);
+  output_file.write(string_stream.str().c_str(), string_stream.str().size());
```

Comment:
You can write something like `output_file << "rename-command KEYS KEYS_NEW\n" << ...` directly.